package.json

```diff
@@ -28,10 +30,10 @@
     "prop-types": "^15.5.7",
     "react-tiny-virtual-list": "^2.0.0",
     "react-transition-group": "^1.1.3",
-    "recompose": "^0.22.0"
+    "recompose": "^0.26.0"
```

Comment:
0.27.0 is the latest on this lib fwiw

Reply:
The ^ will cause 0.26.0 to be used but good point